src/renderer/components/+nodes/nodes.tsx

```diff
@@ -97,7 +97,7 @@ export class Nodes extends React.Component<Props> {
   }
 
   renderMemoryUsage(node: Node) {
-    const metrics = nodesStore.getLastMetricValues(node, ["memoryUsage", "memoryCapacity"]);
+    const metrics = nodesStore.getLastMetricValues(node, ["workloadMemoryUsage", "memoryAllocatableCapacity"]);
```

Comment:
This be misleading when `(memoryAllocatableCapacity - memoryCapacity) < (memoryUsage - workloadMemoryUsage)`. I considered revising it, but I am pressed for time.

Reply:
What do you think it should be instead during when that is true?

Reply:
I think my first comment had a mistake (was rushing to catch a train). When `memoryCapacity - memoryAllocatableCapacity < memoryUsage - workloadMemoryUsage` Then it probably makes more sense to compare `memoryUsage - (memoryCapacity - memoryAllocatableCapacity)` to `memoryAllocatableCapacity`.

That probably seems a bit like magic math, so to be clear: 
`(memoryCapacity - memoryAllocatableCapacity)` represents the portion of memory that is reserved for non-workloads (kubelet, etc.). If those “non-workloads” exceed the reserved amount (or there was no reservation at all) then they begin to consume `memoryAllocatableCapacity`.

Reply:
Ah that makes sense. But can be done in the future.

Reply:
Agreed, the other charts should provide sufficient clarity, and would be a small change.